sdk/core/Azure.Core/src/RequestContent.cs

```diff
@@ -105,7 +111,8 @@ public override void WriteTo(Stream stream, CancellationToken cancellationToken)
                     {
                         cancellationToken.ThrowIfCancellationRequested();
                         var read = _stream.Read(buffer, 0, buffer.Length);
-                        if (read == 0) { break; }
+                        if (read == 0)
```

Comment:
nit: Not sure if this made formatting better

Reply:
I blame auto-format 😃